src/Controller/Game/RegionController.php

```diff
@@ -25,25 +58,14 @@
     public function attack(Request $request, int $regionId): Response
     {
         $player = $this->getPlayer();
-        $em = $this->getEm();
-
-        $region = $em->getRepository('Game:WorldRegion')
-            ->find($regionId);
+        $region = $this->getWorldRegionByIdAndPlayer($regionId, $player);
 
         if (!$region) {
```

Comment:
This acts to anything that is considered 'falsy' by PHP? This also looks like a blacklist, what is the interface that you expect from this method? Can we assert that instead? Like `if (!$region instanceof WorldRegion) {`